src/ice/agent.rs

```diff
@@ -2178,4 +2195,36 @@ mod test {
 
         buf
     }
+
+    #[test]
+    fn relayed_candidates_across_ip_versions_have_lower_priority() {
```

Comment:
I think I want to change this test to actually use two `IceAgent`s and see what ends up getting nominated. That seems like a more robust test to me because it doesn't rely so much on what the internals look like.